sonarts-core/src/rules/noIdenticalFunctionsRule.ts

```diff
@@ -67,8 +67,7 @@ export class Rule extends tslint.Rules.AbstractRule {
   }
 
   private static message(functionBlock: ts.Block): string {
-    const lineOfOriginalFunction =
-      lineAndCharacter(functionBlock.parent!.getStart(), functionBlock.getSourceFile()).line + 1;
+    const lineOfOriginalFunction = startLineAndCharacter(functionBlock.parent!).line + 1;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Forbidden non null assertion ([typescript:S2966](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS2966))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&branch=issue-489&issues=AWGz_GixUhrvXJpox_la&open=AWGz_GixUhrvXJpox_la)

[ISSUE_KEY]: AWGz_GixUhrvXJpox_la